app-games/src/main/java/com/aptoide/android/aptoidegames/AptoideApplication.kt

```diff
@@ -142,7 +143,14 @@ class AptoideApplication : Application(), ImageLoaderFactory, Provider {
       }
     }
     AptoideMMPCampaign.init(BuildConfig.OEMID, BuildConfig.MARKET_NAME)
-    initAds()
+    if ((BuildConfig.MARKET_NAME == "aptoide-games-hmd"
```

Comment:
we already have a function to filter bundles called .filterHMD(). We should generalize it because the rules might change, or we might need to add another model and we should avoid having to change it on multiple places. 
So i would suggest creating a lazy var for example with this rule, which would be called isHMD for example, and then could be used on the filtering of the bundles and here.